### PR TITLE
Draft: Some fixes for `LeTraceurSnork/admin-changelog-md`

### PR DESCRIPTION
**Pull request by gomzyakov**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -15,8 +15,8 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '7.3'
-          tools: composer:v1
+          php-version: '7.4'
+          tools: composer:v2
```
Comment (gomzyakov) [inline .github/workflows/release.yml:19]: - It makes no sense to support PHP 7.3
- Composer 2 is much faster

File: changelog.php [hunk 2]
```diff
@@ -4,14 +4,14 @@
  * @version 1.0.0
  *
  * Plugin Name: Admin CHANGELOG.md
```
Comment (gomzyakov) [inline changelog.php:6]: It looks like it's a plugin for management `CHANGELOG.md`-files.

Maybe use simple name, like a `Show CHANGELOG.md in admin-panel`?

File: composer.json [hunk 3]
```diff
@@ -1,24 +1,20 @@
 {
-    "name": "spectrumdata/admin-changelog-md",
+    "name": "LeTraceurSnork/admin-changelog-md",
     "description": "SpectrumData WordPress plugin \"Admin CHANGELOG.md page\"",
     "type": "wordpress-plugin",
-    "license": "proprietary",
```
Comment (gomzyakov) [inline composer.json:5]: This will be very confusing for most users.

File: readme.txt [hunk 4]
```diff
@@ -10,8 +10,6 @@ Use this plugin to show CHANGELOG.md content inside a WordPress-admin (useful fo
 
 == Changelog ==
 
-= 1.0.0 - 4/20/2022 =
+= 0.1.0 - 4/20/2022 =
```
Comment (gomzyakov) [inline readme.txt:13]: The release `1.0.0` will be after you are sure that the solution is production-ready ;)

File: composer.json [hunk 5]
```diff
@@ -1,24 +1,20 @@
...
+    "license": "MIT",
     "authors": [
         {
             "name": "LeTraceurSnork",
             "email": "le_traceur_snork@mail.ru"
-        },
-        {
-            "name": "Gomzyakov.A",
-            "email": "gomzjakov.a@avtocod.ru"
```
Comment (gomzyakov) [inline composer.json:13]: Thank you, but contributors can be viewed in the section https://github.com/LeTraceurSnork/admin-changelog-md/graphs/contributors

File: composer.json [hunk 6]
```diff
@@ -1,24 +1,20 @@
...
         }
     ],
     "config": {
         "vendor-dir": "vendor"
     },
     "autoload": {
         "psr-4": {
-            "SpectrumData\\Plugin\\Changelog\\": "classes/Changelog"
```
Comment (gomzyakov) [inline composer.json:21]: `Plugin` is an extra word, it is not necessary

File: .github/workflows/release.yml [hunk 7]
```diff
@@ -34,6 +34,7 @@ jobs:
           echo "::set-output name=CHANGELOG::$CHANGELOG"
       - name: Create GH release
         id: create_gh_release
+        # TODO Look at error: Validation Failed: {"resource":"Release","code":"already_exists","field":"tag_name"}
```
Comment (gomzyakov) [inline .github/workflows/release.yml:37]: If you created the release and the tag manually (https://github.com/LeTraceurSnork/admin-changelog-md/releases/tag/v1.0.0) then of course there will be an error - such a tag already exists.

File: .github/workflows/tests.yml [hunk 8]
```diff
@@ -0,0 +1,68 @@
+name: WordPress
```
Comment (gomzyakov) [inline .github/workflows/tests.yml:1]: This file added only for experiments with CI. Please, resolve thread after read it.

File: .github/workflows/release.yml [hunk 9]
```diff
@@ -15,8 +15,8 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
```
Comment (gomzyakov) [inline .github/workflows/release.yml:16]: We have thousands of CI examples for `wordpress-plugin` https://github.com/topics/wordpress-plugin


Maybe just look at other examples?

File: changelog.php (hunk 2 above)
Comment (LeTraceurSnork) [inline changelog.php:6]:
```suggestion
 * Plugin Name: CHANGELOG.md in admin-panel
```

**Pull request merged**